tabular/src/autogluon/tabular/features/feature_metadata.py

```diff
@@ -121,7 +121,7 @@ def get_features(self, valid_raw_types: list = None, valid_special_types: list =
             If special_types is None, then any feature will satisfy the special type requirement (including those with no special types).
         required_exact : bool, default False
             If True, then if a feature does not have the exact same special types (with no extra special types) as required_special_types, it is pruned.
-            This also applied to required_raw_special_pairs if specified.
+            This is also applied to required_raw_special_pairs if specified.
             Has no effect if required_special_types and required_raw_special_pairs are None.
```

Comment:
I'd also clarify here the fact that: columns with object dtype will be ignored by all models.

Reply:
I don't think FeatureMetadata should have comments talking about model implementations, as it is used in various contexts.